packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -135,504 +142,553 @@ const initState = (zcf, manager, idInManager) => {
   });
 };
 
-/** @param {ImmutableState & MutableState} state */
-const constructFromState = state => {
-  /** @type {ImmutableState} */
-  const { idInManager, manager, zcf } = state;
-
-  // #region Computed constants
-  const collateralBrand = manager.getCollateralBrand();
-  const debtBrand = manager.getDebtBrand();
+// #region Computed constants
+const collateralBrand = ({ state }) => state.manager.getCollateralBrand();
```

Comment:
please move the behavior functions within the object that will hold them

Reply:
I'm unclear what you mean by "the behavior functions".   Did you mean inline _all_ the behavior functions or just the two that you highlighted in this comment?  Because they weren't all inlined before, by any means.

Reply:
I did mean all of them. They weren't all inlined in object creation but they were all nested and that's a property that I want to maintain. They also all had JSDoc before and the new object construction breaks that unless they're inlined.

Reply:
Done!  Please give it a look.

Reply:
Perfect, thank you.